### PR TITLE
Update FirebasePerformance.podspec.json for FirebasePerformance 7.3.0

### PR DESCRIPTION
**Pull request by jeremyjiang-dev**

### CODE REVIEW

**Review by paulb777 — COMMENTED** [review]
LGTM, please wait for #7077 to merge before merging this

**Review by visumickey — COMMENTED** [review]
Changes look good to me!

**Review by paulb777 — APPROVED** [review]

**Comment by adozenlines** [comment]
this no longer works 'pod Firebase/Performance' and Xcode reports import FirebasePerformance - not found for - 7.3.0

**Pull request merged**
